installer/macos/build_dmg.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Is it possible to bundle the sscans.app and editor.app into the same dmg?

Reply:
Yes it is possible. Pushed the changes requested.